src/devices/Pigpio/Driver.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Device.Gpio;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Iot.Device.Pigpio
```

Comment:
I don't feel strongly about it but feels to me perhaps Iot.Device.Gpio + class rename would work better and more consistent to System.Device.Gpio